modules/lazy-images/js/lazy-images.js

```diff
@@ -12,15 +12,19 @@ var jetpackLazyImagesModule = function( $ ) {
 		image,
 		i;
 
-	$( document ).ready( function() {
-		lazy_load_init();
 
+
+	lazy_load_init();
+
+	var bodyEl = document.querySelector( 'body' );
+	if ( bodyEl ) {
 		// Lazy load images that are brought in from Infinite Scroll
-		$( 'body' ).bind( 'post-load', lazy_load_init );
+		bodyEl.addEventListener( 'post-load', lazy_load_init );
 
 		// Add event to provide optional compatibility for other code.
-		$( 'body' ).bind( 'jetpack-lazy-images-load', lazy_load_init );
-	} );
+		bodyEl.addEventListener( 'jetpack-lazy-images-load', lazy_load_init );
```

Comment:
**Note:** This is a potential regression.

I just realized these are jQuery events and we can't simply hook into those using `addEventListener`. We'll probably need to `dispatch` regular events instead of using `jQuery.trigger` in code that actually creates these events. I'll need to look into this a bit more.

Reply:
This has been addressed in https://github.com/Automattic/jetpack/pull/14886/commits/cff56d8e2f2a773a9b2df405a2545608991b8cff and the solution is explained in a comment above https://github.com/Automattic/jetpack/pull/14886/files#r390905356